src/ResourceManager/EventGrid/Commands.EventGrid/help/Update-AzureRmEventGridSubscription.md

```diff
@@ -0,0 +1,399 @@
+---
+external help file: Microsoft.Azure.Commands.EventGrid.dll-Help.xml
+Module Name: AzureRM.EventGrid
+online version: 
```

Comment:
@kalyanaj for these new markdown files, be sure to provide the following value for `online version`:

`https://docs.microsoft.com/en-us/powershell/module/azurerm.eventgrid/<cmdlet_name>`

This will enable the `Get-Help <cmdlet_name> -Online` scenario for users so they can jump to the https://docs.microsoft.com page corresponding to the cmdlet.